app/routes/register-your-interest/index.js

```diff
@@ -0,0 +1,66 @@
+const urlPrefix = require('../../config/index').urlPrefix
+const ruralPaymentsAgency = require('../../config/index').ruralPaymentsAgency
+const defraIdConfig = require('../../config').authConfig.defraId
+const BUSINESS_EMAIL_SCHEMA = require('../../schemas/business-email.schema.js')
+const { sendDefraIdRegisterYourInterestMessage } = require('../../messaging/register-your-interest')
+const sendEmail = require('../../lib/email/send-email')
+const { registerYourInterest } = require('../../config').notifyConfig.emailTemplates
+const Joi = require('joi')
+
+const ERROR_MESSAGE = {
+  enterYourEmailAddress: 'Enter your business email address',
+  enterYourEmailAddressInCorrectFormat: 'Enter your email address in the correct format, like name@example.com'
+}
+
+module.exports = [
+  {
+    method: 'GET',
+    path: `${urlPrefix}/register-your-interest`,
+    options: {
+      auth: false,
+      handler: async (request, h) => {
+        if (defraIdConfig.enabled) {
+          return h.view('defra-id/register-your-interest/index', { ruralPaymentsAgency })
+        } else {
+          return h.view('register-your-interest/index', { ruralPaymentsAgency })
+        }
+      }
+    }
+  },
+  {
+    method: 'POST',
+    path: `${urlPrefix}/register-your-interest`,
+    options: {
+      auth: false,
+      validate: {
+        payload: Joi.object({
+          emailAddress: BUSINESS_EMAIL_SCHEMA
+            .messages({
+              'any.required': ERROR_MESSAGE.enterYourEmailAddress,
+              'string.base': ERROR_MESSAGE.enterYourEmailAddress,
+              'string.empty': ERROR_MESSAGE.enterYourEmailAddress,
+              'string.email': ERROR_MESSAGE.enterYourEmailAddressInCorrectFormat
+            })
+        }),
+        failAction: async (request, h, error) => {
+          const errorMessages = error
+            .details
+            .reduce((acc, e) => ({ ...acc, [e.context.label]: { text: e.message } }), {})
+          return h.view(
+            'defra-id/register-your-interest/index',
+            {
+              ...request.payload,
+              ruralPaymentsAgency,
+              errorMessages
+            }
+          ).code(400).takeover()
+        }
+      }
+    },
+    handler: async (request, h) => {
+      await sendEmail(registerYourInterest, request.payload.emailAddress)
```

Comment:
We can use this new template which is a registration of interest confirmation V2 (new for defra id)-> 

https://www.notifications.service.gov.uk/services/4e30070f-25e2-40df-8add-e4375408638f/templates/8c6d9dc1-127d-4355-a16c-cc5fd522f64f

ID is 8c6d9dc1-127d-4355-a16c-cc5fd522f64f